lib/pleaserun/detector.rb

```diff
@@ -25,14 +25,19 @@ def detect_platform
 
   def detect_systemd
     # Expect a certain directory
-    return false unless File.directory?("/lib/systemd/system")
+    return false unless File.directory?("/lib/systemd/system") || File.directory?("/usr/lib/systemd/system")
```

Comment:
CentOS does have /usr/lib/systemd/system and /lib is a symlink to usr/lib, so either check would suffice.
I assume RHEL and Fedora do the same symlinking.